internal/libs/testhelpers/ocr3runner.go

```diff
@@ -100,7 +100,16 @@ func (r *OCR3Runner[RI]) RunRound(ctx context.Context) (result RoundResult[RI],
 
 	// check that all the outcomes are the same.
 	if countUniqueOutcomes(outcomes) > 1 {
-		return RoundResult[RI]{}, fmt.Errorf("outcomes are not equal")
+		decodedOs := make([]exectypes.Outcome, len(outcomes))
```

Comment:
Was failing with non-equal outcomes when we were not sorting by seqNumbers, bug showed when added multiple reports for same chain. This is just logging for future references if something like this happen again.